packages-internal/test-utils/package.json

```diff
@@ -37,8 +37,9 @@
     "@babel/runtime": "^7.24.7",
     "@emotion/cache": "^11.11.0",
     "@emotion/react": "^11.11.4",
-    "@testing-library/dom": "^10.2.0",
+    "@testing-library/dom": "10.2.0",
     "@testing-library/react": "^16.0.0",
+    "@testing-library/user-event": "^14.5.2",
```

Comment:
Touching this updated `@testing-library/dom`'s version in the lockfile, and version `10.3` of that library made many tests fail. I don't have the bandwidth to debug that at the moment, so I pinned it to `10.2` for now.

Reply:
The version has been increased to `10.3.1` on `next` and I just merged the change in this branch, let's see if it works with that version.